tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,332 @@
-                        start, stop)
+evoked_diff = mne.combine_evoked([aud_evoked, -vis_evoked], weights='equal')
+evoked_diff.pick_types('mag').plot_topo(color='r', legend=False)
 
 ##############################################################################
-# Save result in stc files:
+# Time-frequency analysis
+# ^^^^^^^^^^^^^^^^^^^^^^^
+#
+# The :mod:`mne.time_frequency` submodule provides implementations of several
+# algorithms to compute time-frequency representations, power spectral density,
+# and cross-spectral density. Here, for example, we'll compute for the
+# auditory epochs the induced power at different frequencies and times, using
+# Morlet wavelets:
 
-stc.save('mne_dSPM_raw_inverse_Aud')
+frequencies = np.arange(7, 30, 3)
+power = mne.time_frequency.tfr_morlet(aud_epochs, n_cycles=2, return_itc=False,
+                                      freqs=frequencies, decim=3)
+power.plot(['MEG 1332'])
 
 ##############################################################################
-# What else can you do?
-# ^^^^^^^^^^^^^^^^^^^^^
-#
-#     - detect heart beat QRS component
-#     - detect eye blinks and EOG artifacts
-#     - compute SSP projections to remove ECG or EOG artifacts
-#     - compute Independent Component Analysis (ICA) to remove artifacts or
-#       select latent sources
-#     - estimate noise covariance matrix from Raw and Epochs
-#     - visualize cross-trial response dynamics using epochs images
-#     - compute forward solutions
-#     - estimate power in the source space
-#     - estimate connectivity in sensor and source space
-#     - morph stc from one brain to another for group studies
-#     - compute mass univariate statistics base on custom contrasts
-#     - visualize source estimates
-#     - export raw, epochs, and evoked data to other python data analysis
-#       libraries e.g. pandas
-#     - and many more things ...
+# Inverse modeling
+# ^^^^^^^^^^^^^^^^
 #
-# Want to know more ?
-# ^^^^^^^^^^^^^^^^^^^
+# Finally, we can estimate the cortical origins of the evoked activity by
+# projecting the sensor data into this subject's :term:`source space`.
+# MNE-Python supports lots of ways of doing this (minimum-norm estimation,
+# dipole fitting, beamformers, etc); here we'll use minimum-norm estimation
+# (MNE) to generate a continuous cortical activation map. To do this we'll
+# need the inverse operator for this subject (the sample data includes one
+# that's been pre-computed), and we'll need to specify a regularization
+# parameter:
 #
-# Browse :ref:`the examples gallery <sphx_glr_auto_examples>`.
-
-print("Done!")
+# .. TODO: compute inverse operator instead of loading?
+
+inverse_operator_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                     'sample_audvis-meg-oct-6-meg-inv.fif')
+inv_operator = mne.minimum_norm.read_inverse_operator(inverse_operator_file)
+# set the regularization parameter (λ²)
+signal_to_noise_ratio = 3.
```

Comment:
signal_to_noise_ratio -> snr

I would actually suggest following the variable names that we use in the rest of the examples and tutorials. While this is indeed a little bit easier to understand, the cost is that users will be confused when they go to other examples and wonder if it was the same thing. You could add "#signal to noise ratio" in an inline comment though

Reply:
I would also use snr as variable name and add a comment as suggested by Mainak